geoportailv3/externs/fuse.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * @fileoverview Externs for fuse.js
+ * @see https://github.com/krisk/Fuse/blob/master/README.md
+ * @externs
+ */
+
+/**
+ * @typedef {{
+ *    keys: (Array.<string>),
```

Comment:
I don't think you need surrounding brackets here. They're only required when you use `|`, as for the other properties.